config/Mainz-MogonNHR/config.yml

```
@@ -13,12 +13,27 @@ repo: "https://github.com/snakemake-workflows/transriptome-differential-expressi
 
 ## Workflow-specific Parameters:
 
-# the reference genome respectively transcriptome, is defined here:
+# The reference genome respectively transcriptome, is defined here:
+# You can provide either 'genome' or 'annotation' reference files locally.
+# If both 'genome' and 'annotation' are available no 'accession' number is needed. 
```

yamllint

[error] trailing spaces

(trailing-spaces)